doc/source/data/transforming-datasets.rst

```diff
@@ -89,28 +89,50 @@ API in Datasets.
 
 Here are the basics that you need to know about UDFs:
 
-* A UDF can be either a function, or if using the :ref:`actor compute strategy <transform_datasets_compute_strategy>`, a :ref:`callable class <transform_datasets_callable_classes>`.
+* A UDF can be either a function, a generator, or if using the :ref:`actor compute strategy <transform_datasets_compute_strategy>`, a :ref:`callable class <transform_datasets_callable_classes>`.
 * Select the UDF input :ref:`batch format <transform_datasets_batch_formats>` using the ``batch_format`` argument.
 * The UDF output type determines the Dataset schema of the transformation result.
 
 .. _transform_datasets_callable_classes:
 
-Callable Class UDFs
-===================
+Types of UDFs
+=============
+.. tabbed:: "Function UDFs"
```

Comment:
before the tab, can you mention a summary statement - ie "There are three types of UDFs that you can use with Ray Data: Function ..."

Reply:
updated